core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -209,6 +210,11 @@ public SystemSessionProperties(
                         "Use preferred write partitioning",
                         featuresConfig.isUsePreferredWritePartitioning(),
                         false),
+                integerProperty(
+                        PREFERRED_WRITE_PARTITIONING_MIN_NUMBER_OF_PARTITIONS,
+                        "Minimum number of partitions to enable preferred write partitioning`",
```

Comment:
Actually, I think the other comment was better:
```
Use preferred write partitioning when the number of written partitions exceeds given threshold
```

Reply:
@sopel39 s version but 

exceeds the configured threshold